apps/fluent-tester/src/TestComponents/Switch/Switch.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { SWITCH_TESTPAGE } from './consts';
+import { View, StyleSheet } from 'react-native';
+import { Switch } from '@fluentui-react-native/switch';
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
container style exists in Common/styles.ts - `settingsPicker` object

Reply:
Replaced it with settingPicker